packages/sdks/web-component/src/lib/descope-wc/DescopeWc.ts

```diff
@@ -610,6 +631,15 @@ class DescopeWc extends BaseDescopeWc {
         sdkResp?.error?.errorDescription || defaultMessage,
         sdkResp?.error?.errorMessage || defaultDescription,
       );
+
+      // E102004 = Flow requested is in old version
+      // E103205 = Flow timed out
+      if (
```

Comment:
I think we should still log, but maybe just with warning if we are able to recover?

Reply:
Also, do we want to have some way to disable this functionality? if the descoper want to handle it by themselves?

Reply:
We are dispatching an error in this case, this was not changed 
The only change is reloading the flow in some scenarios

Regarding the option to disable it, 
I think we have too many flags already, but we can add another one
Let's decide how we should call it and what should be the default value and i'll add it